frontend/src/pages/Organizations/Organizations.tsx

```diff
@@ -46,28 +46,33 @@ export const Organizations: React.FC = () => {
                 // TODO: use a batch call here instead.
                 const createdOrganizations = [];
                 for (const result of results) {
-                  createdOrganizations.push(
-                    await apiPost('/organizations/', {
-                      body: {
-                        ...result,
-                        // These fields are initially parsed as strings, so they need
-                        // to be converted to arrays.
-                        ipBlocks: (
-                          (result.ipBlocks as unknown as string) || ''
-                        ).split(','),
-                        rootDomains: (
-                          (result.rootDomains as unknown as string) || ''
-                        ).split(','),
-                        tags: ((result.tags as unknown as string) || '')
-                          .split(',')
-                          .map((tag) => ({
-                            name: tag
-                          }))
-                      }
-                    })
-                  );
+                  try {
+                    createdOrganizations.push(
+                      await apiPost('/organizations/', {
+                        body: {
+                          ...result,
+                          // These fields are initially parsed as strings, so they need
+                          // to be converted to arrays.
+                          ipBlocks: (
+                            (result.ipBlocks as unknown as string) || ''
+                          ).split(','),
+                          rootDomains: (
+                            (result.rootDomains as unknown as string) || ''
+                          ).split(','),
+                          tags: ((result.tags as unknown as string) || '')
+                            .split(',')
+                            .map((tag) => ({
+                              name: tag
+                            }))
+                        }
+                      })
+                    );
+                  } catch (e: any) {
```

Comment:
Will this output the error back to the screen or just in the console?

Reply:
Currently, this will only output the error in the console.